lib/global_id/locator.rb

```diff
@@ -126,22 +149,27 @@ def normalize_app(app)
       @locators = {}
 
       class BaseLocator
-        def locate(gid)
-          gid.model_class.find gid.model_id
+        def locate(gid, options = {})
+          model_class = gid.model_class
+          model_class = model_class.includes(options[:includes]) if model_class.respond_to?(:includes)
```

Comment:
We should only call includes if `options[:includes]` is passed. We should not care if `model_class` respond to includes. If someone passed the option, they know it responds to it.
```suggestion
          model_class = model_class.includes(options[:includes]) if options[:includes]
```